source/gems/dry-component/booting-a-dependency.html.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Booting a Dependency
 layout: gem-single
+name: dry-component
```

Comment:
I wonder if there's any way to infer this name from the directory path? We know it'll be `source/gems/<gem name>/...`. It might be nice not to have to require this extra line of boilerplate in every file?